### PR TITLE
EPMDJ-6735: Add card

### PR DESCRIPTION
**Pull request by IliaSolovev**
Issue: EPMDJ-6735

### CODE REVIEW

File: src/components/card/card.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  children: string;
+  href: string;
+  altTextForIcon: string;
+}
+
+export const Card = ({
+  children, iconUrl, href, altTextForIcon,
+}: Props) => (
+  <a href={href} className={styles.card}>
+    <img src={useBaseUrl(iconUrl)} alt={altTextForIcon} />
```
Comment (RomanDavlyatshin) [inline src/components/card/card.tsx]: Do not use `` tag for things that are not **_content_**
Consider implementing icons via any alternative tag (`<i>`, `<span>` ) and use `background-image` css property

File: src/components/card/card.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import styles from './card.module.scss';
+
+interface Props {
+  iconUrl: string;
+  children: string;
+  href: string;
+  altTextForIcon: string;
```
Comment (RomanDavlyatshin) [inline src/components/card/card.tsx]: That prop is not necessary (see comment on `` usage bellow)

File: src/components/card/card.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+export const Card = ({
+  children, iconUrl, href, altTextForIcon,
+}: Props) => (
+  <a href={href} className={styles.card}>
+    <img src={useBaseUrl(iconUrl)} alt={altTextForIcon} />
+    {children}
+    <img
```
Comment (RomanDavlyatshin) [inline src/components/card/card.tsx]: That should not be `` as well

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]

**Review by RomanAchilov — APPROVED** [review]

File: src/components/card/card.tsx [hunk 4]
```diff
@@ -7,19 +7,20 @@ interface Props {
   iconUrl: string;
   children: string;
   href: string;
-  altTextForIcon: string;
 }
 
 export const Card = ({
-  children, iconUrl, href, altTextForIcon,
+  children, iconUrl, href,
```
Comment (RomanDavlyatshin) [inline src/components/card/card.tsx]: Rename `iconUrl` -> `iconSrc` (and use `src` everywhere).

**Review by RomanDavlyatshin — APPROVED** [review]

**Pull request merged**